### ISSUE TITLE
SplitButton: menu item is using 'aria-level' attribute incorrectly

### ISSUE THREAD

**Issue by ppc-harmoney** [primefaces/primeng#15130]
### Describe the bug

We are using prime ng in an application that is legally required to be WCAG compliant. To check compliance we are using [Axe devtools](https://www.deque.com/axe/browser-extensions/#:~:text=Overview-,Overview,your%20time%20with%20false%20positives.). 

It appears that the menu items for the split button are not setting the correct aria attributes, as we are seeing this error: 
`ARIA attribute is not allowed: aria-level="1"`

![image](https://github.com/primefaces/primeng/assets/84312273/dc7f98d4-2837-4d3e-a150-c509fbd42f18)

### Environment

Windows 11 Pro

### Angular version

16.2.9

### PrimeNG version

16.9.1

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

18.18.2

### Browser(s)

chrome 123

### Steps to reproduce the behavior

Open the menu of a splitbutton and run a scan using axe devtools.

### Expected behavior

I'm not entirely sure if the attribute should be corrected or removed, but axe devtool should not indicate it as a problem.
